workers/approov-token-binding/README.md

```diff
@@ -0,0 +1,174 @@
+# Approov Token Binding Quick Start
+
+This quick start is for developers familiar with Cloudflare who are looking for a quick intro into how they can add [Approov](https://approov.io) into an existing project. Therefore this will guide you through the necessary steps for adding Approov to an existing site being routed through Cloudflare.
```

Comment:
Vraious comments in here relevant to the standard token binding. Can't we structure this one as additonal steps over basic token test integration to avoid duplicate content?